Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2548,6 +2557,11 @@ - (BOOL)_isPictureInPictureActive
 #endif
 }
 
+- (BOOL)_isInWindowActive
+{
+    return NO;
```

Comment:
Are we going to fix this in a later patch?

Reply:
Yes!